sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -131,11 +131,12 @@ def read(
         # type: (...) -> Dict[str, Any]
         """Read the container properties.
 
+        :param Any args: The explicit arguments of the method.
```

Comment:
We actually shouldn't be documenting this, as it's only the overload that shows up in the reference docs, not this base method signature - and so the presence of args will look a bit strange:
<img width="535" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/8689453/dcd95f0f-1176-4a41-9a91-44915719bc8f">